RabbitMQModule doesn't have forRoot and forRootAsync methods

Hi. I'm trying to implement custom message parsing and serializing feature for RabbitMQ module because I would like to exchange messages in XML format.

So far I have made necessary changes and trying to launch `integration/rabbitmq` example, but it doesn't have `@golevelup/nestjs-rabbitmq` dependency in `package.json` and it's also using forRootAsync method for module setup which doesn't exists.

I assume there is minor inconsistency between packages and I would like to create PR to fix it. Could you provide additional information about this?

@jmcdo29 The issue is the typescript declarations are incorrect so any TS build will fail. The declaration as of `1.7.1` looks like 
```typescript
import { DiscoveryService } from '@golevelup/nestjs-discovery';
import { DynamicModule, OnModuleInit } from '@nestjs/common';
import { ExternalContextCreator } from '@nestjs/core/helpers/external-context-creator';
import { AmqpConnection } from './amqp/connection';
import { RabbitMQConfig } from './rabbitmq.interfaces';
declare const RabbitMQModule_base: import("../../modules/lib").IConfigurableDynamicRootModule<RabbitMQModule, RabbitMQConfig>;
export declare class RabbitMQModule extends RabbitMQModule_base implements OnModuleInit {
    private readonly discover;
    private readonly amqpConnection;
    private readonly externalContextCreator;
    private readonly logger;
    constructor(discover: DiscoveryService, amqpConnection: AmqpConnection, externalContextCreator: ExternalContextCreator);
    static build(config: RabbitMQConfig): DynamicModule;
    static attach(connection: AmqpConnection): DynamicModule;
    onModuleInit(): Promise<void>;
}
export {};
```

Causing 
```bash
../../../node_modules/@golevelup/nestjs-rabbitmq/lib/rabbitmq.module.d.ts:6:43 - error TS2307: Cannot find module '../../modules/lib'.
: 6 declare const RabbitMQModule_base: import("../../modules/lib").IConfigurableDynamicRootModule<RabbitMQModule, RabbitMQConfig>;
:                                             ~~~~~~~~~~~~~~~~~~~
: src/modules/entity-publisher/entity-publisher.module.ts:21:24 - error TS2339: Property 'forRootAsync' does not exist on type 'typeof RabbitMQModule'.
: 21         RabbitMQModule.forRootAsync({
:                           ~~~~~~~~~~~~
```

This line in particular is referencing a wrong path 
```
import("../../modules/lib").IConfigurableDynamicRootModule<RabbitMQModule, RabbitMQConfig>;
``` 
there is no `modules/lib` when installing this package in a project. its actually `nestjs-modules/lib` since its depdendent on the `nestjs-modules` package

--- a/packages/rabbitmq/src/rabbitmq.module.ts
+++ b/packages/rabbitmq/src/rabbitmq.module.ts
@@ -1,11 +1,19 @@
 import { DiscoveryModule, DiscoveryService } from '@golevelup/nestjs-discovery';
-import { createConfigurableDynamicRootModule } from '@golevelup/nestjs-modules';
+import {
+  createConfigurableDynamicRootModule,
+  IConfigurableDynamicRootModule
+} from '@golevelup/nestjs-modules';
 import { DynamicModule, Logger, Module, OnModuleInit } from '@nestjs/common';
 import { ExternalContextCreator } from '@nestjs/core/helpers/external-context-creator';
 import { groupBy } from 'lodash';
 import { AmqpConnection } from './amqp/connection';
 import { RABBIT_CONFIG_TOKEN, RABBIT_HANDLER } from './rabbitmq.constants';
 import { RabbitHandlerConfig, RabbitMQConfig } from './rabbitmq.interfaces';
+
+declare var placeholder: IConfigurableDynamicRootModule<
+  RabbitMQModule,
+  RabbitMQConfig
+>;
 
 @Module({
   imports: [DiscoveryModule]

